pull_request_template.md

```diff
@@ -0,0 +1,60 @@
+# Resolved Issues
+
+# Description
+
+# Review Checklists
+
+Please check each item **before approving** the pull request. While going
+through the checklist, it is recommended to leave comments on items that are
+referenced in the checklist to make sure that they are reviewed. If there are
+multiple reviewers, copy the checklists into sections titled `## [Reviewer Name]`.
+If the PR doesn't touch Solidity and/or Rust, the corresponding checklist can
+be removed.
+
+## [[Reviewer Name]]
+
+### Solidity
+
+- [ ] **Tokens**
+    - [ ] Do all `approve` calls use `forceApprove`?
+    - [ ] Do all `transfer` calls use `safeTransfer`?
+    - [ ] Do all `transferFrom` calls use `msg.sender` as the `from` address?
+        - [ ] If not, is the function access restricted to prevent unauthorized
+              token spend?
+- [ ] **Low-level calls**
```

Comment:
Can we define what a low-level call is?

Reply:
Sure. I'll add this in parentheses. A low-level call is one of `call`, `delegatecall`, `staticcall`, or `callcode`. `transfer` and `send` are also considered low-level calls by some.

Reply:
Looks like `callcode` was deprecated (good fucking riddance), so I'll remove that.